### PR TITLE
Added guidance for configuring Cloudsmith as a private registry

### PR DESCRIPTION
**Pull request by ciaracarey**
Added guidance for configuring Cloudsmith as a private registry
Cloudsmith is a Cloud Native Artifact Management Platform that supports 30+ formats and integrates with Dependabot. 

Cloudsmith's support for Dependabot has been tested and we have documented it on our site: https://help.cloudsmith.io/docs/dependabot.

### Why:

Closes: 

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Comment by nguyenalex836** [comment]
@ciaracarey Thanks so much for opening a PR! I'll get this triaged for review ✨

**Comment by nguyenalex836** [comment]
@ciaracarey Thank you for opening a PR! Per our “[About contributing to GitHub Docs](https://docs.github.com/en/contributing/collaborating-on-github-docs/about-contributing-to-github-docs#pull-requests)” article -

> We only document GitHub products, features, tools, and extensions. We may mention or link to third-party tools to demonstrate how a feature works, but we do not accept pull requests to document third-party tools or integrations unless they were codeveloped with GitHub.

For this reason, we will be closing this PR. If you would like to update our docs with content outside of third-party tools or integrations, feel free to open another PR or issue 💛

**Comment by ciaracarey** [comment]
thanks @nguyenalex836
could I rewrite the PR with a reduced mention of Cloudsmith?

A few other artifact management tools were mentioned on this page, so I thought it would be helpful to our joint customers.

**Comment by nguyenalex836** [comment]
@ciaracarey I'll check with our team to see if this would be acceptable, and will provide an update as soon as I have more info!

File: content/code-security/dependabot/working-with-dependabot/guidance-for-the-configuration-of-private-registries-for-dependabot.md [hunk 1]
```diff
@@ -621,6 +622,38 @@ The Azure Artifacts password must be an unencoded token and should include a `:`
...
+    directory: "/"
+    schedule:
+      interval: "daily"
+    registries:
+      - cloudsmith
+    commit-message:
+      prefix: "deps"
+    open-pull-requests-limit: 10
+```
+
```
Comment (jurre) [inline content/code-security/dependabot/working-with-dependabot/guidance-for-the-configuration-of-private-registries-for-dependabot.md:656]: This seems like just a normal registry configuration for `pip`, no need to add an explicit example for it, I think if we just mention that Cloudsmith is supported that should be sufficient for users to figure out how to set it up.

Comment (ciaracarey) [reply content/code-security/dependabot/working-with-dependabot/guidance-for-the-configuration-of-private-registries-for-dependabot.md:656]: Will I make a new PR, just mentioning Cloudsmith?

Thanks again for reviewing this

Comment (nguyenalex836) [reply content/code-security/dependabot/working-with-dependabot/guidance-for-the-configuration-of-private-registries-for-dependabot.md:656]: Closing this PR given the incoming new PR 💛

**Pull request closed without merging**